### PR TITLE
Generate test_results.tsv for fasttest

### PR DESCRIPTION
**Pull request by novikd**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):

- Build/Testing/Packaging Improvement

### CODE REVIEW

**Review by tavplubix — APPROVED** [review]

**Comment by alesapin** [comment]
I'll ask to add this image into "available" list.

**Comment by alesapin** [comment]
@mergifyio update

**Comment by novikd** [comment]
@Mergifyio update

File: docker/images.json [hunk 1]
```diff
@@ -41,6 +41,12 @@
         "name": "clickhouse/pvs-test",
         "dependent": []
     },
+    "docker/test/util": {
```
Comment (azat) [inline docker/images.json:44]: @novikd Shouldn't fasttest image be dependent from docker/test/util?

**Pull request merged**